cmd/main.go

```diff
@@ -78,19 +84,22 @@ func run(options commandOptions) error {
 }
 
 func buildCmdTree() *cobra.Command {
-	if internal.ConfiguredAndV1() {
-		internal.PrintV1DeprecationWarning()
+	if internal.IsConfigured() && !internal.IsVersionSupported() {
+		fmt.Printf(noticeColor, "The v1 projects are deprecated since Feb 1, 2020.\n"+
+			"See how to upgrade your project to v2: https://book.kubebuilder.io/migration/guide.html\n")
 	}
 
 	// kubebuilder
 	rootCmd := newRootCmd()
 
 	// kubebuilder alpha
 	alphaCmd := newAlphaCmd()
-	// kubebuilder alpha webhook (v1 only)
-	if internal.ConfiguredAndV1() {
-		alphaCmd.AddCommand(newWebhookCmd())
+
+	// kubebuilder update (v1 only)
+	if !internal.IsVersionSupported() {
+		rootCmd.AddCommand(newUpdateCmd())
 	}
+
```

Comment:
Commads are added in alphabetical order, `Update` should be last.

Reply:
I did not change the current order and I am not quite sure about the role `Commands are added in alphabetical order, Update should be last`. See that the order was not changed ate all:

<img width="1334" alt="Screenshot 2020-02-15 at 14 04 02" src="https://user-images.githubusercontent.com/7708031/74589316-19b70880-4ffc-11ea-92cc-d34087ac38fa.png">

Reply:
You did change the order, newUpdateCmd was added the last and you are adding it first.

Your IDE is comparing the alpha subcommand with the update subcommand.

Reply:
Oh. I see what you mean now. Good spot. 
Change has done and closing this one as solved.